docs/configuration/sinks/mattermost.rst

```diff
@@ -75,7 +81,7 @@ Save the file and run
 
     helm upgrade robusta robusta/robusta --values=generated_values.yaml
 
-You should now get playbooks results in Mattermost!
+You should now get playbooks results in Mattermost! Make sure to add the bot to the required channels.
```

Comment:
`Make sure to add the bot to the required channels.` is only if you use non-admin bot?

Reply:
Ah yeah it is not required for messages, but if you have a non-admin bot you need the bot to be added in order to post files. I'll update accordingly